### PR TITLE
Add scalability test setup script

### PR DESCRIPTION
**Pull request by kashifest**
This PR adds the scalability test setup script which needs the ips of the VMs and then goes and sets up the whole test bed.

### CODE REVIEW

File: tools/scalability/setup_vms.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+wait
+
+master_vm_ip=$(cat vm_ip_list.txt | head -n 1 )
+
+for ip in $ips;do 
+	
+	if [[ $ip == $master_vm_ip ]]; then
+		ssh ubuntu@$ip -- "mkdir bm_crs"
+      	continue
+    fi
```
Comment (furkatgofurov7) [inline tools/scalability/setup_vms.sh:39]: indentation
```suggestion
        fi
```

File: tools/scalability/vm_ip_list.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+188.95.226.181
```
Comment (furkatgofurov7) [inline tools/scalability/vm_ip_list.txt:1]: please add some note that you are expecting the first IP in the list to be the `master VM` IP

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
Good job!
Some nits inline:

File: tools/scalability/vm_ip_list.txt (hunk 2 above)
Comment (kashifest) [reply tools/scalability/vm_ip_list.txt:1]: Done

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**